### PR TITLE
New Meat Cooking Temperature Cheat Sheet

### PR DESCRIPTION
**Pull request by DavidWittman**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [x] New Instant Answer
    - [x] Cheat Sheets: **`New {Cheat Sheet Name} Cheat Sheet`**
    - [ ] Other: **`New {IA Name} Instant Answer`**
- [ ] Improvement
    - [ ] Bug fix: **`{IA Name}: Fix {Issue number or one-line description}`**
    - [ ] Enhancement: **`{IA Name}: {Description of Improvements}`**
- [ ] Non–Instant Answer
    - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**

###### Description of changes

A new cheat sheet providing internal cooking temperatures for meat, as provided by the Food Network.

###### Which issues (if any) does this fix?

Solves https://duck.co/ideas/idea/4899/temperature-guidelines-cooking-meat

###### People to notify (@mention interested parties)

N/A

---

Instant Answer Page: https://duck.co/ia/view/meat_cooking_temperature_cheat_sheet

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @mention

### CODE REVIEW

**Comment by daxtheduck** [comment]
@DavidWittman Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please head to https://quackslack.herokuapp.com/ to get an invite.

>More Info: http://docs.duckduckhack.com/submitting/submitting-overview.html

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by DavidWittman** [comment]
I had to re-create my IA instant answer page to un-pluralize `temperature`. I edited my PR description to reflect this change.

**Comment by edgesince84** [comment]
your example queries are all one big query - they'll need to be separated on the IA page.

File: share/goodie/cheat_sheets/json/meat-cooking-temperature.json [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    "sourceUrl": "http://www.foodnetwork.com/recipes/articles/meat-and-poultry-temperature-guide.html"
+  },
+  "aliases": [
+    "meat temperature",
+    "chicken temperature", "chicken cooking temperature",
+    "poultry temperature", "poultry cooking temperature",
+    "turkey temperature", "turkey cooking temperature",
+    "beef temperature", "beef cooking temperature",
+    "lamb temperature", "lamb cooking temperature",
+    "pork temperature", "pork cooking temperature"
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/meat-cooking-temperature.json]: Lets add `meat internal temperature`

**Comment by GuiltyDolphin** [comment]
@DavidWittman Might it not be better to go for the USDA temperatures? As it says on the source page:

*"Out of a concern for safety, the USDA recommends higher temperatures for whole cuts of beef and lamb than we do. We have included the USDA recommendations, leaving it up to you to decide."*

File: share/goodie/cheat_sheets/json/meat-cooking-temperature.json (hunk 1 above)
Comment (DavidWittman) [inline share/goodie/cheat_sheets/json/meat-cooking-temperature.json]: Good idea. Added. :+1:

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=meat temperature cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/meat_cooking_temperature_cheat_sheet_mobile.png?Q709lML4eH)

**Comment by DavidWittman** [comment]
> Might it not be better to go for the USDA temperatures (only a couple of fields different)? 

Thanks for pointing this out @GuiltyDolphin. I was debating how to handle that. I've been using the "Food Network" temperatures my whole life and have managed to not poison myself, but that's purely subjective.

I'm open to any advice on how you think it'd be best to format this in the cases where the USDA temperatures differ. I can just suffix each entry with *USDA Recommends X* or something.

**Comment by DavidWittman** [comment]
@edgesince84 I fixed the query examples. Thank you.

**Comment by edgesince84** [comment]
it looks good to me!

**Pull request merged**